com.regnosys.rosetta/src/com/regnosys/rosetta/Rosetta.xtext

```diff
@@ -74,7 +74,22 @@ ShortcutDeclaration:
 Condition:
 	'condition' RosettaNamed? ':' Definable?
 		Annotations?
-		(expressions+=RosettaCalcExpression ';')+
+		( constraint = OneOf
```

Comment:
Can we add a validation warning if multiple `one-of`s are used on a `data` object

Reply:
SureI will create a new issue to collect validation rules